docs/performance.rst

```diff
@@ -0,0 +1,77 @@
+Performance
+===========
+
+LSDB is a high-performance package built to support the analysis of large-scale astronomical datasets.
+One of the performance goals of LSDB is to add as little overhead over the input-output operations as possible.
+We achieve this aim for catalog cross-matching, spatial and data filtering operations by using
+the `HiPSCat <https://github.com/astronomy-commons/hipscat>`_ data format,
+efficient algorithms,
+and `Dask <https://dask.org/>`_ framework for parallel computing.
+
+Here we demonstrate the results of the performance tests of LSDB for cross-matching operations,
+performed on `Bridges2 cluster at Pittsburgh Supercomputing Center <https://www.psc.edu/resources/bridges-2/>`_ 
+using a asingle node with 128 cores and 256 GB of memory.
+
+Cross-matching performance overhead
+-----------------------------------
+
+We compare I/O speed and cross-matching performance of LSDB on an example cross-matching of
+ZTF DR14 (metadata only, 1.2B rows, 60GB)
```

Comment:
metadata only - what does that mean here

Reply:
"Object" catalog, it is "metadata" in ZTF DR language. Would it be better to say "object" catalog?